.firebase/hosting.YnVpbGQ.cache

```diff
@@ -1,26 +1,26 @@
-static/media/watch.d89e9882.svg,1583835579195,ad83f2247d18d7b851fbc9ec745dc605fab74ac2fd497658b29a75d043700dee
-static/media/world.0c2ba674.svg,1583835579195,0a7015ae144dffae074f5ca59d91b2fffa77a8f69ac9ad9899240ada3b1f24e8
+asset-manifest.json,1584103995318,87cd1e32220180dd78d8a8fb15b773bd62a4e0d3a2b8316882155bef3d7b182d
+index.html,1584103995318,b0f6203df9734536d7c3b3667d229efecf39e4907cad277bc35c1cbdaa6d45da
+manifest.json,1584103974670,341d52628782f8ac9290bbfc43298afccb47b7cbfcee146ae30cf0f46bc30900
+robots.txt,1584103974670,391d14b3c2f8c9143a27a28c7399585142228d4d1bdbe2c87ac946de411fa9a2
+service-worker.js,1584103995318,adc70191a35cbd3496bceda092707aa7b33bed65f46a887ecbd5c58faf5a052f
+static/js/2.c6432189.chunk.js.LICENSE.txt,1584103995325,fc49ea4371db68c951af2e8b12b5cd9018c6a265901c711cc48aeeaecccc371c
+static/js/runtime-main.f446ce33.js,1584103995314,6940d6b21704243ba818a659adbf7e38e5070c12d9858b43ee8db7a9ed9e9ca8
+static/media/app_pg_one_bg.a3ed93ea.svg,1584103995313,e7ca0b575a203840bd582ea4e1afc57cd971dc3dcc8b42b403f3ba69db0769cd
+static/media/book.1f21e0ad.svg,1584103995315,4ef8470d1e71337f976a7acee778942b25d2370117e891d598bec7971731eeb5
+static/media/connect.0a853296.svg,1584103995313,0c372faa9f42bc553bff283845be18aa8483eab9b8733a59d705c5c012317b3b
+static/js/runtime-main.f446ce33.js.map,1584103995325,645767864e3f9def8f430fe7a1ed9ec7092108867ae43e093b16ccd6e8f98b0e
+precache-manifest.0dd037f390b17718271b7c5433c639af.js,1584103995318,9740ea5247307bf1fc73262250c16fc0b3c7ceb6a0948c9f9947d18e4f5c47e8
+static/media/bubble.ed7c7fa5.svg,1584103995315,2386cbd017fcee4c1bbefa84c22bed4dd62a2c506b12202d48c442f45f873b8e
+static/js/main.76d5ab1c.chunk.js,1584103995313,e62743e3909d6527ffa2a2cdc3d3c3b52c0122e47db3a77940ba9b4a864c1625
+static/media/depth.331d059a.svg,1584103995313,d3b91f97e0575ac6e94b8fea8f99244133decfb20d945260b33da77e4fcad5c5
+static/media/goggles.bc203d3a.svg,1584103995313,391e842b8dd37abd579d2ec28e596d164a840f32c1fcf0f3d761f56138f2385e
+static/media/home.02770c65.svg,1584103995314,df89d4413e154d5dae590344aefcff929d1ac6262b2ca28c12ff65474d6db8fd
+static/media/heart.096ab24e.svg,1584103995315,76a479fd5819b130c3dadcdf8e2baadb47ec116c19921deaa478c5aec3a07e44
+static/media/mark.9eaf0e6d.svg,1584103995311,6b32fa0e8bd440eb09ecfa3951f25f6fc890e1c653e1ff6344a183e0d541bfa6
+static/media/lupe.9e601ee6.svg,1584103995314,3089d54a19cb6d66d5411f37a0277057a50c480fbec6b1bf4a8f93dcc3e45ea5
+static/media/watch.d89e9882.svg,1584103995313,ad83f2247d18d7b851fbc9ec745dc605fab74ac2fd497658b29a75d043700dee
+static/media/world.0c2ba674.svg,1584103995314,0a7015ae144dffae074f5ca59d91b2fffa77a8f69ac9ad9899240ada3b1f24e8
+static/js/main.76d5ab1c.chunk.js.map,1584103995325,e08a8c9282cb938d261ce23a07d86773e573ec94da3a4b7d480351c3f86d85b1
+static/js/2.c6432189.chunk.js,1584103995318,e66ea65250938925fff6e6bc05104708e5ba898f5578c119b323573d04bce207
+static/media/portrait.8e605b52.jpg,1584103995314,476ae1d70614af75a57689ef595d68a03c49d140fff68979db53659b9641898d
```

Comment:
Bin nicht sicher, aber könnte mir vorstellen, dass es auch Sicherheitsgründen sinnvoll ist, die Datei in gitignore zu packen?